config/initializers/devise.rb

```diff
@@ -8,7 +8,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  #config.secret_key = 'c9aac5e1283cb3302f0a7a2217dbd287a8b8564070b519d96246666ba172191d3b1267d1b578c0e1771890a15fe2ef30edbd3adc3cac38d6b9a2b918b8421ab9'
+  # config.secret_key = 'c9aac5e1283cb3302f0a7a2217dbd287a8b8564070b519d96246666ba172191d3b1267d1b578c0e1771890a15fe2ef30edbd3adc3cac38d6b9a2b918b8421ab9'
```

Comment:
Rubocop doesn't seem to grok this intentionally. I don't see any configuration that would allow a comment in this scenario to not have a leading space. I also don't see any GitHub issues complaining about this, which perhaps indicates that people agree in general with leading spaces before comments everywhere.

For example, the default config files generated by Rails adhere to the leading space before comments everywhere. Even the Devise config where this offense was flagged has leading spaces everywhere else, so the lack of space on this line is an anomaly with respect to the rest of the file. On their master branch, even the secret_key has a leading space, so either this was a copy and paste error, or it used to be a bug in Devise which has now been fixed. See: https://github.com/plataformatec/devise/blob/0c1b1a896f549f92d76baf06fcece150420e94bf/lib/generators/templates/devise.rb

Hence, I vote to fix this.